### ISSUE TITLE
[Question] Base_heght_l2 dynamic target height issue

### ISSUE THREAD

**Issue by jaykorea** [isaac-sim/IsaacLab#1546]
### Question

Basic questions, related to robot learning, that are not bugs or feature requests will be closed without reply, because GitHub issues are not an appropriate venue for these.

Advanced/nontrivial questions, especially in areas where documentation is lacking, are very much welcome.

For questions that are related to running and understanding Isaac Sim, please post them at the official [Isaac Sim forums](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/isaac_sim_forums.html).

on the base_height_l2 reward function, should the reward function consider the ground height like below?

    if sensor_cfg is not None:
        sensor: RayCaster = env.scene[sensor_cfg.name]
        # Adjust the target height using the sensor data
        adjusted_target_height = target_height + torch.mean(sensor.data.ray_hits_w[..., 2], dim=1)
        
        but origin reward function looks like this 
        adjusted_target_height = target_height + sensor.data.pos_w[:, 2]
        
       'sensor.data.pos_w[:, 2]' appears to set to the sensor position in z-axis on world frame. it does not count the ground height

**Comment by RandomOakForest** [comment]
Thanks for posting this. This has been tagged for fixing this in an upcoming release.
